tests/solana_tests/using.rs

```diff
@@ -38,6 +38,40 @@ fn using_for_contracts() {
 
     assert_eq!(runtime.logs, "Hello");
 
+    let mut runtime = build_solidity(
+        r#"
+        interface I {
+            function f(int) external;
+        }
+
+        library L {
+            function F(I i, bool b, int n) public {
+                if (b) {
+                    print("Hello");
+                }
+            }
+        }
```

Comment:
Would you mind not creating a function that receives an interface?
This test will stop working because of #1517.

Reply:
Please, do not create a giant monolithic test. It is better to create a new function whose name explains exactly what you are testing.